cabal.project

```diff
@@ -37,15 +37,17 @@ optional-packages: vendored/*/*.cabal
 tests: true
 
 package *
-  ghc-options: -haddock
+  -- ghc 8.10 cannot build ghc-lib 9.2 with --haddock
+  -- ghc-options: -haddock
   test-show-details: direct
 
 write-ghc-environment-files: never
 
-index-state: 2022-03-08T10:53:01Z
+index-state: 2022-04-27T09:22:49Z
 
 constraints:
-    hyphenation +embed
+    hyphenation +embed,
+    hlint +ghc-lib
```

Comment:
How does this affect our ability to install from Hackage? Do we need a comment here? "Remove when there is a release of hlint that uses this by default"?

Reply:
This change removes the need to have per-ghc-version flags, enabling a single `cabal.project` file. And when `hlint` sets this flag by default, it will enable `cabal install haskell-language-server` across all ghc versions.
I'll add a comment.